codegen/src/type_gen.rs

```diff
@@ -209,7 +218,11 @@ fn parse_types() -> BTreeMap<u32, Type> {
         }
 
         let element = match &*kind {
-            "R" => range_elements[&oid],
+            "R" => match &*raw_type["typtype"] {
+                "r" => range_elements[&oid],
+                "m" => multi_range_elements[&oid],
```

Comment:
You can find the description of this field in the `pg_type` catalog table (https://www.postgresql.org/docs/current/catalog-pg-type.html). The relevant part is:
> r for a range type, or m for a multirange type.